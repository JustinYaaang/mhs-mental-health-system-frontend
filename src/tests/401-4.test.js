import React from 'react'
import { shallow } from 'enzyme'
import Error401 from '../layouts/401'
import Error404 from '../layouts/404'
import { configure } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
configure({ adapter: new Adapter() })

describe('401', () => {
  it('should render correctly', () => {
    const component = shallow(<Error401 />)

    expect(component).toMatchSnapshot()
  })
})



describe('404', () => {
  it('should render correctly', () => {
    const component = shallow(<Error404 />)

    expect(component).toMatchSnapshot()
  })
})


