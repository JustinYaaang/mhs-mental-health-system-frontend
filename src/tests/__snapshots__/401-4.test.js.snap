// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`401 should render correctly 1`] = `
<div
  class="page-body"
>
  <div
    class="page-wrap"
  >
    <h1>
      401
    </h1>
    <h2>
      unauthorized
    </h2>
    <p>
      Well, what to say. Hmm? Sorry ...
    </p>
    <p
      class="redirect-p"
    >
      <a
        class="redirect-a"
        href="/"
      >
        home
      </a>
    </p>
  </div>
</div>
`;

exports[`404 should render correctly 1`] = `
<div
  class="page-body"
>
  <div
    class="page-wrap"
  >
    <h1>
      404
    </h1>
    <h2>
      not found
    </h2>
    <p>
      Sorry this page does not exist ...
    </p>
    <p
      class="redirect-p"
    >
      <a
        class="redirect-a"
        href="/"
      >
        home
      </a>
    </p>
  </div>
</div>
`;
